proxyfsd/saioproxyfsd0.conf

```diff
@@ -13,6 +13,8 @@ WhoAmI: Peer0
 .include ./httpserver.conf
 .include ./debug.conf
 
+.include ./proxyfs-evtlog.conf
+
 # put it here to have the last word (override previous)
-#[Logging]
-#TraceLevelLogging: inode fs headhunter
+[Logging]
+TraceLevelLogging: headhunter inode sortedmap
```

Comment:
I assume this was because you had instrumented sortedmap...
All those traces are gone now (good thing), so I guess we could delete "sortedmap" here, right?